src/backend/distributed/utils/multi_partitioning_utils.c

```diff
@@ -186,48 +193,17 @@ fix_partition_shard_index_names(PG_FUNCTION_ARGS)
 	EnsureCoordinator();
 
 	Oid relationId = PG_GETARG_OID(0);
-
-	Relation relation = try_relation_open(relationId, AccessExclusiveLock);
-
-	if (relation == NULL)
-	{
-		ereport(NOTICE, (errmsg("relation with OID %u does not exist, skipping",
-								relationId)));
-		PG_RETURN_VOID();
-	}
-
-	if (relation->rd_rel->relkind != RELKIND_PARTITIONED_TABLE)
-	{
-		relation_close(relation, NoLock);
-		ereport(ERROR, (errmsg(
-							"Fixing shard index names is only applicable to partitioned"
-							" tables, and \"%s\" is not a partitioned table",
-							RelationGetRelationName(relation))));
-	}
+	Oid parentIndexOid = InvalidOid; /* fix all the indexes */
 
 	if (!IsCitusTable(relationId))
 	{
-		relation_close(relation, NoLock);
-		ereport(ERROR, (errmsg("fix_partition_shard_index_names can "
-							   "only be called for distributed partitioned tables")));
+		ereport(ERROR, (errmsg("fix_partition_shard_index_names can only be called "
+							   "for distributed or Citus local partitioned tables")));
```

Comment:
"distributed or Citus local partitioned tables" might be a bit confusing

maybe "Citus tables"?